### PR TITLE
DB-6429 Test coverage for Diablo

### PR DESCRIPTION
**Pull request by liprais**
added several configs to enable code coverage analysis using jacoco.
in order to generate a code coverage report, run 'mvn clean install -Dmaven.test.failure.ignore=true -Djacoco -Pmem,core,cdh5.12.0'

### CODE REVIEW

File: start-splice-cluster [hunk 1]
```diff
@@ -188,9 +188,9 @@ echo "Starting Master and 1 Region Server. Log file is ${SPLICE_LOG}"
 ## (master + region server on 1527)
 
 if [ -z "${SERVER_SSLMODE}" ]; then
-  (${MVN} exec:exec -P${PROFILE},spliceFast ${SYSTEM_PROPS} -Dxml.plan.debug.path=${DEBUG_PATH} > ${SPLICE_LOG} 2>&1) &
+  (${MVN} exec:exec -P${PROFILE},spliceFast,jacoco  ${SYSTEM_PROPS} -Dxml.plan.debug.path=${DEBUG_PATH} > ${SPLICE_LOG} 2>&1) &
```
Comment (OlegMazurov) [inline start-splice-cluster]: The 'jacoco' mode should not be used unconditionally in every standalone run.

File: start-splice-cluster [hunk 2]
```diff
@@ -188,9 +188,9 @@ echo "Starting Master and 1 Region Server. Log file is ${SPLICE_LOG}"
...
 else
-  (${MVN} exec:exec -P${PROFILE},spliceFastSSL ${SYSTEM_PROPS} -Dxml.plan.debug.path=${DEBUG_PATH} > ${SPLICE_LOG} 2>&1) &
+  (${MVN} exec:exec -P${PROFILE},spliceFastSSL,jacoco ${SYSTEM_PROPS} -Dxml.plan.debug.path=${DEBUG_PATH} > ${SPLICE_LOG} 2>&1) &
```
Comment (OlegMazurov) [inline start-splice-cluster]: Ditto.

File: start-splice-cluster (hunk 1 above)
Comment (liprais) [reply start-splice-cluster]: only when '-Djacoco' and -Pjacoco both provided ,the code coverage anlysis will be enabled ,so it won’t be used unconditionally in every standalone run, when '-Djacoco' is not provided, the jacocoeagent will be '-server',which is harmless I think

Comment (OlegMazurov) [reply start-splice-cluster]: How are you going to provide '-Djacoco' for start-splice-cluster? 
Why do you need '-Pjacoco' in start-splice-cluster at all?

Comment (liprais) [reply start-splice-cluster]: You are right,It is unnecessary  for code coverage I have removed it

**Pull request merged**